layouts/partials/scripts.html

```diff
@@ -2,8 +2,9 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
 {{ $jsBase := resources.Get "js/base.js" }}
+{{ $jsAnchor := resources.Get "js/anchor.js" }}
```

Comment:
I'm not sure if this is correct. What does the second part of $jsSearch do?

Reply:
Hrmm, I didn't write that bit but as far as I can see it's because search.js has Go template stuff (ie stuff in {{ }}) in it so we need to render it with ExecuteAsTemplate before using it. Your script doesn't so you can just use it "as is" (ie you are right!)

See: https://regisphilibert.com/blog/2018/07/hugo-pipes-and-asset-processing-pipeline/#customizing-our-sass-variables-with-executeastemplate